.github/workflows/pages.yaml

```diff
@@ -17,18 +21,24 @@ jobs:
       - name: Configure pages
         uses: actions/configure-pages@v5
 
-      - name: Setup python
+      - name: Install uv
+        uses: astral-sh/setup-uv@v3
+        with:
+          enable-cache: true
+          cache-dependency-glob: pyproject.toml
+          cache-suffix: docs
+
+      - name: Setup Python
         uses: actions/setup-python@v5
         with:
-          python-version: "3.x"
-          cache: "pip"
+          python-version-file: pyproject.toml
 
-      - name: Install dependencies
-        run: pip install .[docs]
+      - name: Install Python dependencies
+        run: uv pip install .[docs]
 
       - name: Generate HTML
         run: |
-          sphinx-build -d _build/doctrees docs/source _build/html
+          uv run sphinx-build -d _build/doctrees docs/source _build/html
```

Comment:
**[Question, non-blocking]** If we're installing packages into the global environment by setting `UV_SYSTEM_PYTHON=1`, do we need to prefix the `sphinx-build` call with `uv` here?